### PR TITLE
Readme3

### PR DESCRIPTION
**Pull request by RyzenDew**
added a warning on piracy
added git clone instruction's 
Fixed a typo

### CODE REVIEW

File: .github/readme.md [hunk 1]
```diff
@@ -12,7 +12,9 @@
 </div>
 
 > **Warning** <br/>
-> It's NOT possible to run any games yet, and there is no ETA for when this will change
+> It's NOT possible to run any games yet, and there is no ETA for when this will change.
+
+> Do not ask for link to games or system files. Piracy is not permitted on the GitHub or in the Discord.
```
Comment (elad335) [inline .github/readme.md:17]: Shouldn't you use nor here?

**Comment by sampletext32** [comment]
maybe you could also fix this typo?
`How to create to create virtual hdd` to
`How to create virtual hdd`

File: .github/readme.md (hunk 1 above)
Comment (RyzenDew) [reply .github/readme.md:17]: i will

Comment (RyzenDew) [reply .github/readme.md:17]: > Shouldn't you use `nor` here?

explain i had a brain fart can you write it out

**Comment by RyzenDew** [comment]
> maybe you could also fix this typo? `How to create to create virtual hdd` to `How to create virtual hdd`

that is done thanks

**Comment by GamingDucking** [comment]
> maybe you could also fix this typo? `How to create to create virtual hdd` to `How to create virtual hdd`

Still there

File: .github/readme.md (hunk 1 above)
Comment (elad335) [reply .github/readme.md:17]: ```Do not ask for link to games or system files. Piracy is not permitted on the GitHub nor in the Discord.```

Comment (RyzenDew) [reply .github/readme.md]: all done

**Comment by RyzenDew** [comment]
it should be good now

**Pull request merged**